README.md

```diff
@@ -29,6 +29,16 @@ Trestle provides tooling to help orchestrate the compliance process across a num
 - Support within trestle to streamline management within a managed git environment.
 - An underlying object model that supports developers interacting with OSCAL artefacts.
 
+## Important Note:
+
+The current version of trestle supports NIST OSCAL 1.0.2 and beyond.  There as a breaking change in OSCAL moving from
```

Comment:
Shall we simply say 1.0.2 as we cannot guarantee the support of the potential 1.0.3?

Reply:
I agree and have removed the "beyond" - and clarified handling of 1.0.0 in import and 1.0.2 on output.